### PR TITLE
Expand Trivy scanning workflow

### PR DESCRIPTION
**Pull request by n-th**
**Pull Request check list**

- [ ] Proper tests/regressions included?
- [ ] Documentation updated?

**Affected functionality**
Changes trivy scan from a docker image to the whole repository.

**Description of change**

**Which issue this PR fixes**

### CODE REVIEW

File: .github/workflows/trivy.yml [hunk 1]
```diff
@@ -31,11 +31,12 @@ jobs:
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@7b7aa264d83dc58691451798b4d117d53d21edfe
         with:
-          image-ref: 'galadriel-doc:${{ github.sha }}'
```
Comment (mchurichi) [inline .github/workflows/trivy.yml:34]: does this remove the dependency on the image?

Comment (n-th) [reply .github/workflows/trivy.yml:34]: We changed the code to scan the vulnerability in our repository instead of running it in the images.

**Review by polimoraes — APPROVED** [review]
- use Trivy for scan the entire github repository 
- include MEDIUM in severity level filter

File: .github/workflows/trivy.yml (hunk 1 above)
Comment (mchurichi) [reply .github/workflows/trivy.yml:34]: awesome, thanks!

**Review by mchurichi — APPROVED** [review]
💯

**Pull request merged**